include/seqan3/argument_parser/validators.hpp

```diff
@@ -485,6 +485,98 @@ struct default_validator
     }
 };
 
+//!\brief A helper struct to chain validators recursively via operator+()
+//!\ingroup validators
+template <validator_concept validator1_type, validator_concept validator2_type>
+    requires std::Same<typename validator1_type::value_type, typename validator2_type::value_type>
+class validator_chain_adaptor
```

Comment:
does it implement the validator_concept?
Shouldn't the doxygen than say `//!\implements ...concept`?

Reply:
:+1: I actually did not do that for any of the validators... I will change all of them